crates/codegen/runtime/cargo/src/runtime/language.rs.jinja2

```diff
@@ -30,7 +30,10 @@ use crate::parser_support::{
 #[cfg_attr(feature = "slang_napi_interfaces", napi(namespace = "language"))]
 pub struct Language {
     {%- if not rendering_in_stubs -%}
-        {%- for version in model.parser.referenced_versions -%}
+        {%- for version in model.breaking_versions -%}
+            {% if loop.first %} {# The first supported version may not be referenced by the items #}
```

Comment:
For more context, it's possible right now to have `enabled = From(THE_FIRST_VERSION)` in the DSL v2, which is what testlang does but what solidity does not.

I don't have a preference whether we should disallow it or not (it probably should be a warning at best if they were supported by proc macros) but I figured that it's not in scope for this PR.

Reply:
I think we should disallow it if it doesn't have additinal semantics/provide a benefit to us. If for any reason, it is to avoid special casing it in various places in codegen (exactly like this here).
But there are definetely more important validation rules we can implement if we decide to spend more time there.